### ISSUE TITLE
Linux-remote ruined firewalld settings

### ISSUE THREAD

**Issue by alvinbengeorge** [firewalld/firewalld#818]
**What happened**:
HI, so recently I installed an app called linux-remote through pip3, I gave it permissions for firewalld.
After I restarted my computer, I could only open Gnome Apps(Gnome Web, Gnome tweaks,Nautilus).
I couldnt open anything like my IDEs or Brave browser

**How to reproduce it (as minimally and precisely as possible)**:
pip3 install linux-remote(DO NOT SUDO as it could bypass through everything, check every single permission its asking)
https://pypi.org/project/linux-remote/
Android app link: https://play.google.com/store/apps/details?id=org.linuxremote.app

**Anything else we need to know?**:
I disabled firewalld, but thats not a solution.
Do you have any setting which says something like "Open only System apps"? 
Please let me know

Thanks in advance

**Comment by erig0** [comment]
Can you check `/var/log/firewalld` for errors? It's possible the package you installed caused invalid configuration.

**Comment by alvinbengeorge** [comment]
```
2021-04-22 21:57:40 ERROR: UNKNOWN_INTERFACE: 'enp1s0f1' is not in any zone
2021-07-05 10:51:40 WARNING: ALREADY_ENABLED: 9212:tcp
2021-07-05 11:10:35 WARNING: ALREADY_ENABLED: 9212:tcp
2021-07-05 11:10:35 WARNING: ALREADY_ENABLED: 9212:udp
```
Does this say anything?
I didnt enable it yet

**Comment by erig0** [comment]
> ```
> 2021-04-22 21:57:40 ERROR: UNKNOWN_INTERFACE: 'enp1s0f1' is not in any zone
> ```

That may be a runtime only error. Not a permanent configuration error.

When you say you "couldnt open anything"... does that mean the programs would not launch? Or does it mean they would launch, but fail to load websites?

**Comment by alvinbengeorge** [comment]
Program would not launch, but i guess that was not the problem, cause now its fixed, its fine now, my firewalld is enabled. Its working. Thank you for asking and helping me out.  
It might have been temporary. Or some bug caused by some update. I dont know about that.
It worked when I restarted it the second time with firewalld disabled
So basically if I tried opening Brave or IDLE/Atom, It wouldnt open, but if I ask it open Epiphany, it will.
And thats how I issued this problem, through epiphany.


Thank you.
Alvin
